README.md

```diff
@@ -15,10 +15,35 @@ To see what's incoming, check the [open pull requests](https://github.com/greene
 Instructions for using Manubot Rootstock for your own manuscript are still evolving.
 The recommended approach is to clone this repository, as detailed [here](https://github.com/greenelab/manubot-rootstock/issues/6#issuecomment-314541837).
 
-## Source
+## Repository directories & files
 
-The manuscript source is located in [`content`](content).
-Text should be written in markdown files, with a digit prefix for ordering (e.g. `01.`, `02.`, etc.) and a `.md` suffix.
+See [`CONTRIBUTING.md`](CONTRIBUTING.md) for documentation on how to create or contribute to a manuscript.
```

Comment:
We could split `CONTRIBUTING.md` so that `CONTRIBUTING.md` contains only the `Manubot feedback` section and a new `USAGE.md` (or similar) contains all of the documentation.